ivy/func_wrapper.py

```diff
@@ -454,6 +454,14 @@ def _inputs_to_ivy_arrays(*args, **kwargs):
         -------
             The return of the function, with ivy arrays passed in the arguments.
         """
+        if not ivy.get_array_mode():
+            x = ivy.nested_map(args, lambda x: ivy.is_native_array(x))
```

Comment:
Lets not add checks for native arrays, I guess its safe to assume that the user will pass native array when array_mode is False, since we halt all conversions of arrays in that case anyway. Its not expected to work with ivy arrays.

Reply:
Yep absolutely, we're free to assume that it's a native array 👍